packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -736,60 +756,41 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
     // voting activity the voters balance is updated accordingly. The caller can provide a maxTraversals parameter to
     // limit the number of resolved requests to traverse in this call to bound the gas used. Note each iteration of
     // this function re-uses a fresh slash variable to produce useful logs on the amount a voter is slashed.
-    function _updateAccountSlashingTrackers(address voterAddress, uint64 maxTraversals) internal {
-        VoterStake storage voterStake = voterStakes[voterAddress];
+    function _updateAccountSlashingTrackers(address voter, uint64 maxTraversals) internal {
```

Comment:
I've further simplified this method by adding two internal methods that we can use now: a) `requestSlashingTrackers` (which we previously had as an external method) which returns slashing trackers and re-uses them here so we dont need to re-call this method/double up the code and b) `getVoterParticipation` which returns an enum for a given voter for their participation.

overall, this decreased the line cound and complexity in this function, while saving a bit of gas.